packages/core/strapi/src/node/staticFiles.ts

```diff
@@ -96,17 +96,18 @@ const writeStaticClientFiles = async (ctx: BuildContext) => {
           : undefined,
     })
   );
+  const prettier = await import('prettier');
```

Comment:
nowdays we can use top-level await so you could also just do that right after the imports to collocate all of these. (considering here lazy loading gvies no gain)

Reply:
It doesn't let us because we also export as cjs. There may be some configuration changes we can make in the build process but I'm not that familiar with it.

Reply:
let's not bother ;)